pkg/monitors/collectd/cpu/metadata.yaml

```diff
@@ -1,6 +1,10 @@
 monitors:
 - dimensions:
   doc: |
+    **This monitor is deprecated in favor of the `cpu` monitor.  Please switch
+    to that monitor, as this monitor will be remove in a future agent
```

Comment:
```suggestion
    to that monitor, as this monitor will be removed in a future agent
```

Reply:
might be useful to just put "removed in 5.0" instead of future

Reply:
> might be useful to just put "removed in 5.0" instead of future

It isn't being removed in 5.0, it is just not part of the default config.  It will be totally removed in some later major release.